lib/galaxy/schema/groups.py

```diff
@@ -0,0 +1,80 @@
+from typing import List
+
+from pydantic import Field
+from typing_extensions import Literal
+
+from galaxy.schema.fields import (
+    DecodedDatabaseIdField,
+    ModelClassField,
+)
+from galaxy.schema.schema import Model
+
+GROUP_MODEL_CLASS = Literal["Group"]
+
+
+class GroupIndexResponse(Model):
+    """Response schema for a group."""
+
+    model_class: GROUP_MODEL_CLASS = ModelClassField(GROUP_MODEL_CLASS)
+    id: DecodedDatabaseIdField = Field(
+        ...,
```

Comment:
It is fine to use the `Required` constant instead of the ellipsis (...) for required fields, the reason the ellipsis was used in all the previous models is because the constant was not available in older versions of Pydantic.